recipes-devtools/opkg/opkg_%.bbappend

```diff
@@ -9,6 +9,8 @@ SRC_URI += " \
             file://run-ptest \
             file://0001-opkg-key-add-keys-even-if-creation-date-is-in-the-fu.patch \
             file://0002-libopkg-clear-curl-properties-on-download-error-to-p.patch \
+            file://0001-opkg_solver_libsolv-Pass-opkg_config-noaction-as-noa.patch \
+            file://0001-pkg_src_verify-Verify-feeds-signatures-only-once.patch \
```

Comment:
`.patch`es are applied to recipe sources in alphanumeric-order. In this case, it doesn't matter because your patches are functionally independent and so don't require any particular ordering. But it's more common that you would renumber the `.patch` file name to explicitly define their sequence. eg.

```bash
file://0003-opkg_solver_libsolv-Pass-opkg_config-noaction-as-noa.patch \
file://0004-pkg_src_verify-Verify-feeds-signatures-only-once.patch \
```

I'll leave it up to you to decide if you want to re-number the patches, but I do ask that you keep the batch of `.patch` files in the bbappend in sorted order, rather than adding your new patches at the end. Keeping them sorted helps other devs reviewing the bbappend to understand the order in which they are going to be applied by bitbake.

Reply:
I renumbered the patches so - it looks cleaner this way :-)